### DISCUSSION TITLE
MEETING: Monthly EMu Users - 09 April 2024

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#1041] [MEETINGS]
# Agenda: Monthly EMu User Meeting
## Anthropology Scheduling count data migration
- JJ will let you know when (a few days to a weeks advanced noticed) but by the end of this month.
### Anthro:  [JK, LH, ED, ]
- [LH] added q’s for online catalog / collections-anthro
- Catalog cards - looking for MM unattached to Cat
- Helpdesk tkt - will likely set up a CSV report in MM to find these

### Botany: [KH]
- IPT-question
- Will poke Matt to respond to email about labelsheet report.
#### Fossil Invertebrates [PM]
- 5 interns June 3 start
#### Paleobotany [MD]
- All good
#### Fossil Verts [WS]
- Working through Parts/Preps w/ CM
#### Herps: [JM]
- All good
#### Fishes: [SM]
- Follow-up q for Parts/Prep, re:logic update for indicators
- A value in prep/part + if count or fragment > 0 would trigger the indicator 
#### IZ: [JDA, MP]
- Working on label report changes; 
- [JJ] once changes are consolidated for IZ reports, submit 1 helpdesk tkt
#### Insects:  [MT]
- Bruno’s grasshopper project - ~5000-10k image import by end of month
- Weds mtg - Elizabeth/Bruno/CIL
#### Mammals: [AWF, LJ]
- [Reports with identifications / ‘cf’ currently doesn’t display in Summary or DwC field](https://github.com/fieldmuseum/EMu-Documentation/discussions/1049)
- [JJ] - share an example IRN - ‘cf’ should display in Qualified names
- Tissue tab mtg question - how publish tissue type/etc info to DwC
- Current extensions [possibly not all available on IPT] - https://rs.gbif.org/extensions.html (see GGBN extensions for tissue-related data)
### Photo Archives: [NC]
- Working on mapping FM/EMu
### Action: [ZK, KK, MJ, FC]
- Questions below

### Exhibitions: [MG]
- ExhObj module access - straightened out for user groups (ExhibitsConservation & others)
- ExhibitsRead group - alternatively, add ExhObj. module read-access to other groups
- Coordinate demo of different user group tabs
- April 23 - demo exhibit objects
- Later emu meetings - demo catalog

### Repatriation [SD, DJ]
- Updates to tracking funerary items – “UFO” (unassociated funerary obj.) & “AFO” (associated funerary obj.) will start displaying in summary string for catalog records where relevant
### Training: [Training docs](https://github.com/fieldmuseum/EMu-Documentation/wiki/admin-TRAINING) | [Signup-sheet](https://docs.google.com/spreadsheets/d/1lnzfEgmmQXwzqB9x895ScfAjS_m5TZ9x4kq0k9twW6Q/edit#gid=1392027855)
- Please be sure to request logins to EMu for all people who have signed up for training. Prior to a scheduled training date, please all interns/staff that you are responsible for know their user names and passwords and have logged into EMu at least once.
#### EMu
[Import training](https://docs.google.com/spreadsheets/d/1lnzfEgmmQXwzqB9x895ScfAjS_m5TZ9x4kq0k9twW6Q/edit#gid=751871139&range=A1)

#### NetX
[NetX Training (2024 tab)](https://docs.google.com/spreadsheets/d/15t-Kn1qmMbZ2lPVo1KJdAqnCt9dbdFT6XzeTTpHnfPY/)
[NetX Working Group Questions and Answers](https://docs.google.com/document/d/1j339qxi6qXkQ9m1c20WIko9C9t5FgEYy54IRgxYOnFg/edit)

#### ecatalogue
- [Preps and Parts](https://github.com/fieldmuseum/EMu-Documentation/discussions/1037)
- All decisions for moving forward with the Prep Type, Count, Part/Reproduce and frag fields have been made
- Global Replace homework: All requested Prep Types have been added to the lookup table. If you haven’t already, please update your prep types and counts.
- Add definitions to field level help for discussed workflows (.e.g how your collection uses the counts fields. How you use individual terms, like whole specimen)

**Comment by rondlg** [comment]
- [Question 44: How/Where to track outreach? Github #1042](https://github.com/fieldmuseum/EMu-Documentation/discussions/1042)
- [Question 45: How to manage image/audio data? [ZK/FC/MJ/KK - Action] Github #1043](https://github.com/fieldmuseum/EMu-Documentation/discussions/1043)
- [Question 46:  What is the approach to assigning multiple  counties to a single locality? [RDLP,JD - IZ] github #1044](https://github.com/fieldmuseum/EMu-Documentation/discussions/1044)
- [Question 47: Possible updates to Anthro online collections page (LH) github #1045](https://github.com/fieldmuseum/EMu-Documentation/discussions/1045)
- [Question 48: read-only access for users: Re Adding to “Exhibits Read-only” one at a time? (Marie Georg) github #1046](https://github.com/fieldmuseum/EMu-Documentation/discussions/1046)

**Comment by fmjjones** [comment]
- [Records with identifications / ‘cf’ currently doesn’t display in Summary or DwC field, #1049](https://github.com/fieldmuseum/EMu-Documentation/discussions/1049)
